frontend/package.json

```diff
@@ -24,11 +24,14 @@
     "**/*": "npx prettier --write . --ignore-unknown"
   },
   "dependencies": {
+    "@date-io/dayjs": "^2.16.0",
```

Comment:
No usage for this package?

Reply:
yes, DatePicker select value is Dayjs type.